interface/wx/font.h

```diff
@@ -322,16 +322,11 @@ class wxFontInfo
     /**
         Constructor setting the font size in points to use.
 
-        The canonical type of @a pointSize argument is @c float, however any
-        other integer type, as well as @c double, is also accepted for
-        compatibility.
-
-        Notice that until wxWidgets 3.1.2, the type could only be @c int.
+        Note that until wxWidgets 3.1.2, the type of @a pointSize was @c int.
```

Comment:
I'd like to restore the old wording:
```suggestion
        Note that until wxWidgets 3.1.2, the type of @a pointSize could only be @c int.
```

Reply:
This puzzles me. It doesn't make sense to say what the type of a _parameter_ "could" be. It was `int`. Period. The type of an _argument_ to that parameter could be anything _convertible to_ `int`, but we don't need to explain conversions. For a template parameter it might make some sense to talk about what type it could be, but there is no template involved here (anymore).

Reply:
Maybe I didn't express it well, but the idea is to let people know that font sizes used to be integer, but can now be floating point.

This is useful for the same reason `@since` annotations are useful: of course you don't care about this if you're only using wx 3.1.2+, but you might be writing code which targets both it and 3.0. Or even if you're using wx 3.6, you might be migrating some old code written for 3.0 and wondering why didn't it use floating point font sizes -- the documentation would give you the explanation.

Reply:
> the idea is to let people know that font sizes used to be integer

Sure, but how does the changed version not do that? It literally says that before 3.1.2 `pointSize` was an `int`.